.github/ISSUE_TEMPLATE/general_issue.md

```diff
@@ -0,0 +1,44 @@
+---
+name: General Issue
+about: If your issue does not fall in any of the other categories create an issue with this template.
+title: "[GENERAL ISSUE]"
+labels: guidance
+assignees: ''
+
+---
+
+<!--
+⚠️ Important Information
+Please look at,
+- [Open issues](https://github.com/aws-cloudformation/cloudformation-guard/issues)
+- All our [PRs](https://github.com/aws-cloudformation/cloudformation-guard/pulls?q=is%3Apr), both open and close
+- Our [documentation](https://github.com/aws-cloudformation/cloudformation-guard#aws-cloudformation-guard) 
+before creating an issue to see if your question was already answered.
+-->
+
+**Describe the issue**
+A clear and concise description of what the issue is.
+<!--
+Make sure you are not falling prey to the [X/Y problem](http://xyproblem.info)!
+-->
+
+**Any examples**
+Please supply:
+
+1. Example rules and template that you have tried
+1. The commands you used to invoke the tool
+1. The output of a `-v` log level if it's not related to cfn-guard-lambda, or the relevant CloudWatch log messages if it is related to the cfn-guard-lambda
+
+**NOTE: Please be sure that the templates, rules and logs you provide as part of your issue do not contain any sensitive information.**
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
tougher to copy-paste to reproduce issues from screenshots, so wonder if we should discourage them in favor of text